tools/triage/batch_invite.py

```diff
@@ -0,0 +1,36 @@
+from github import Github
+import os
+
+token = os.environ.get("TOKEN")
+organization_name = os.environ.get("ORG_NAME")
+
+def main():
+
+    with open("../names.txt") as names_file:
+        names = names_file.readlines()
+
+    total = 0
+
+    for name in names:
+        try:
+            gh = Github(token)
+            org = gh.get_organization(organization_name)
+            ghuser = gh.get_user(name.strip())
+            ghlablovers = org.get_team_by_slug("Labrador-Lovers")
+
+            try:
+                org.invite_user(ghuser, teams=[ghlablovers])
```

Comment:
I am not sure this is a script for triagers rather than github org admins.

Reply:
could just be an admin folder rather than triage?